appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockComponent.java

```diff
@@ -855,41 +849,6 @@ public final SourceStructureExplorerItem getSourceStructureExplorerItem() {
     return sourceStructureExplorerItem;
   }
 
-  /**
-   * Returns the asset node with the given name.
-   *
-   * @param name  asset name
-   * @return  asset node found or {@code null}
-   */
-  protected ProjectNode getAssetNode(String name) {
-    Project project = Ode.getInstance().getProjectManager().getProject(editor.getProjectId());
-    if (project != null) {
-      HasAssetsFolder<YoungAndroidAssetsFolder> hasAssetsFolder =
-          (YoungAndroidProjectNode) project.getRootNode();
-      for (ProjectNode asset : hasAssetsFolder.getAssetsFolder().getChildren()) {
-        if (asset.getName().equals(name)) {
-          return asset;
-        }
-      }
-    }
-    return null;
-  }
-
-  /**
-   * Converts the given image property value to an image url.
-   * Returns null if the image property value is blank or not recognized as an
-   * asset.
-   */
-  protected String convertImagePropertyValueToUrl(String text) {
```

Comment:
Is this code removed the same as the one in `MocksComponentsUtil`? Any risk of breaking anything?

Reply:
Yes, this method and `MockComponentsUtil.convertAssetValueToUrl` essentially did the same thing. I have updated all the usages of this method so there shouldn't be any breakages.